.github/sync.yml

```diff
@@ -4,8 +4,13 @@ group:
       18f/methods
       18f/ux-guide
     files:
+      - .github/workflows/sync_templates.yml
       - .github/sync.yml
-      - .github/ISSUE_TEMPLATE/
-      - .github/ISSUE_TEMPLATE.md
-      - .github/PULL_REQUEST_TEMPLATE/
-      - .github/PULL_REQUEST_TEMPLATE.md
+      - source: .github/ISSUE_TEMPLATE/
+        deleteOrphaned: true
+      - source: .github/ISSUE_TEMPLATE.md
+        deleteOrphaned: true
+      - source: .github/PULL_REQUEST_TEMPLATE/
+        deleteOrphaned: true
+      - source: .github/PULL_REQUEST_TEMPLATE.md
+        deleteOrphaned: true
```

Comment:
Turning these into objects allows me to set the `deleteOrphaned` key. They were just strings before.